next.json

```diff
@@ -1050,6 +1050,8 @@
                 "name": "configure",
                 "args": [
                     {"name": "format", "type": "texture format"},
+                    {"name": "allowedUsage", "type": "texture usage bit"},
+                    {"name": "initialUsage", "type": "texture usage bit"},
```

Comment:
Does this guarantee the usage bit at the start of the subpass? I remember still needing to include an explicit transition.